From cd183c5b05201ac3385ae847aecf070b1162372a Mon Sep 17 00:00:00 2001
From: "Pawse, GuruprasadX" <guruprasadx.pawse@intel.com>
Date: Fri, 13 Jan 2017 10:49:09 +0530
Subject: [PATCH 0241/1240] ASoC: Intel: Add Icelake machine id to use RT700

Icelake supports RT700 SoundWire codec as well, hence updating
the machine id list.

Change-Id: I2ff08cac04d4affcdae92a36fc4b74f352cb624c
Signed-off-by: Pawse, GuruprasadX <guruprasadx.pawse@intel.com>
Reviewed-by: R, Dharageswari <dharageswari.r@intel.com>
Reviewed-by: Kale, Sanyog R <sanyog.r.kale@intel.com>
Reviewed-by: Singh, Guneshwor O <guneshwor.o.singh@intel.com>
Reviewed-by: Prodduvaka, Leoni <leoni.prodduvaka@intel.com>
Reviewed-by: Kp, Jeeja <jeeja.kp@intel.com>
Tested-by: Sm, Bhadur A <bhadur.a.sm@intel.com>
---
 sound/soc/intel/skylake/skl.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/sound/soc/intel/skylake/skl.c b/sound/soc/intel/skylake/skl.c
index 8e7e362..101312a 100644
--- a/sound/soc/intel/skylake/skl.c
+++ b/sound/soc/intel/skylake/skl.c
@@ -1059,7 +1059,11 @@ static const struct sst_acpi_mach sst_cnl_devdata[] = {
 };
 
 static struct sst_acpi_mach sst_icl_devdata[] = {
+#if IS_ENABLED(CONFIG_SND_SOC_RT700)
+	{ "dummy", "icl_rt700", "intel/dsp_fw_icl.bin", NULL, NULL, NULL },
+#else
 	{ "dummy", "icl_rt274", "intel/dsp_fw_icl.bin", NULL, NULL, NULL },
+#endif
 	{}
 };
 
-- 
2.7.4

